tests/models/test_agent.py

```diff
@@ -38,3 +43,43 @@ def test_get_agent_by_name_not_found(db_session):
     db_session.commit()
 
     assert get_agent_by_name(db_session, 'different') is None
+
+
+def test_list_all_agents_size(db_session):
+    db_session.add(Agent(name='test'))
+    db_session.commit()
+
+    assert len(list_all_agents(db_session)) == 1
+
+    db_session.add(Agent(name='test2'))
+    db_session.commit()
+
+    assert len(list_all_agents(db_session)) == 2
+
+
+def test_agent_connection_status(db_session, test_client):
+    agent = Agent(name='test')
+    db_session.add(agent)
+    db_session.flush()
+    agent_id = agent.id
+    db_session.commit()
+
+    websocket_manager = WebSocketManager()
+
+    @app.websocket_route('/websocket_connect')
+    async def connect(websocket: WebSocket):
```

Comment:
No real reason to create a route and use `test_client` for this. You can just mock the `WebSocket`. I can also update the test in `AP-12` though.

Reply:
Alright, i'll leave it as it is then